content/tutorials/Team13Tutorial.md

```diff
@@ -0,0 +1,244 @@
+
+Pros:
+1. Provides real-time feedback on battery life.
+2. Prevents damage to the battery by avoiding overcharging or over-discharging.
+3. Offers insights to maximize battery life and performance.
+
+Cons:
+
+1. The more complex the circuit, the higher the cost (e.g., precise resistors may be required).
+2. Limited accuracy, as voltage alone isn’t the only factor that determines the charge state.
+
+Key Concepts:
+
+1. Voltage Divider
+2. Resistor Ratios with Multiple Batteries
+3. Arduino Basics
+
+## Getting Started
+
+You will receive an ESP32 board and USB-C to USB-C connector cable. The ESP32 board utilizes the ESP32 (obviously). The ESP32 is a system on a chip with Wifi and bluetooth capacbilities. This means that it can be used to make IoT projects(Internet of things). However, we will be using it very simply, you do not need to understand everything it can do for this tutorial.
+
+You will also receive a breadboard, jumper cables, and resistors. The rows of the breadboard are connected while the columns are not. The resistors and jumper cables should fit snuggly into the holes of the breadboard. 
+
+We will be using the Arduino IDE (Integrated Development Environment). This is a free, open source program that allows users to write code and upload it to boards. 
+
+### Required Downloads and Installations
+
+If you don't have the Arduino IDE already, download it [here](https://www.arduino.cc/en/software).
+
+### Required Components
+
+List your required hardware components and the quantities here.
+
+| Component Name | Quanitity |
+| -------------- | --------- |
+|     ESP32      |     1     |
+| ESP32 Connector Cable | 1  |
+|Resistors (47kΩ)|     2     |
+| 18650 Battery  |     1     |
+| Battery Holder |     1     |
+|Mini Breadboard |     1     |
+| Jumper Cables  |     2     |
+
+
+### Required Tools and Equipment
+
+Computer, Arduino IDE
+
+## Part 01: Setting up the Circuit
+
+### Introduction
+
+In this section, we will be discussing the voltage divider and circuit setup. We will not be writing any code, so a computer is not yet needed.
+
+### Objective
+
+- Understand how to pick values for a voltage divider.
+- Understand how to properly connect the battery, resistors, and ESP32.
+
+### Background Information
+
+Give a brief explanation of the technical skills learned/needed
+in this challenge. There is no need to go into detail as a
+separation document should be prepared to explain more in depth
+about the technical skills
+
+If you've taken a look at the battery we gave you, you might notice that it says 3.7V on it. The maximum voltage input for the ESP32 is 3.3V, so we need to find a way to decrease the voltage from the battery, we can do this with a <b>voltage divider</b>.
+
+A voltage divider is a passive linear circuit made up of two resistors, although sometimes more are used to get a specific resistance value. 
+![Voltage Divider Schematic and Equation](Team13Photos/voltagedivider.png)
```

Comment:
the ECE 196 website background is basically black, and this image is black strokes on a transparent background so it is not visible. here's what it looks like:

<img width="810" alt="image" src="https://github.com/user-attachments/assets/dccc1f0e-50e7-43c8-b4a7-a3d9afacfd2e">